### PR TITLE
Issue#1108 last movie details

### LINKED ISSUE #1108: Update movie cards & movie banners
https://projects.invisionapp.com/d/main#/console/18708905/389334924/preview
Movie banners should go from

to this

https://projects.invisionapp.com/d/main#/console/18708905/389334925/preview
Movie cards from

to 

- [x] Production status label 
- [ ] Sales agent logo
- [x] Country of origin
- [x] Length
- [x] Genres
- [x] Add to wishlist icon
- [x] Remove production year

### PR DESCRIPTION
**Pull request by ioaNikas**
- [x] Minor enhancements on movie banners informations
- [x] Catalogue list view now have access to wishlist functions
- [x] New icon in header to display count of wishlist items
- [x] Updated catalogue home header content and style
- [x] Deleted most of titles of svg icons

close #1108

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html [hunk 1]
```diff
@@ -29,16 +30,20 @@ <h4>{{ section.subline}} </h4>
                     <mat-icon svgIcon="heart"></mat-icon>
                   </button>
                 </ng-template>
-                <div fxLayout="column" fxLayoutAlign="center start" class="informations">
-                  <b *ngIf="movie.main.status as status" class="status-text">{{ status | translateSlug: 'MOVIE_STATUS'}}</b>
+                <div fxLayout="column" fxLayoutAlign="center start" fxLayoutGap="5px" class="informations">
+                  <b *ngIf="movie.main.status as status" class="status-text">{{ status.toUpperCase() }}</b>
```
Comment (phpgeekfr) [inline apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: no more `translateSlug` ?

File: apps/catalog/marketplace/marketplace/src/app/movie/search/search.component.html [hunk 2]
```diff
@@ -278,26 +278,28 @@ <h2>Search</h2>
...
               <mat-card-content class="table-size-text">
-                <b *ngFor="let country of movie.main.originCountries | slice:0:2; let isLast=last">
-                  {{ country | translateSlug: 'TERRITORIES' }}{{isLast ? '' : ', '}}
+                <b *ngFor="let country of movie.main.originCountries | slice:0:2; let isLast=last; let i=index">
+                  {{ country | translateSlug: 'TERRITORIES' }}{{isLast ? '' : ', '}}{{i == 1 && movie.main.originCountries.length > 2 ? ',  ' : ''}}
                 </b>
                 <b *ngIf="movie.main.originCountries.length > 2">... </b>
-                <b *ngIf="movie.main.length">| {{ movie.main.length }} min </b><b>| {{ movie.main.genres[0] | translateSlug: 'GENRES'}}</b>
+                <b *ngIf="movie.main.length">| {{ movie.main.length }} mins. </b><b>| {{ movie.main.genres[0] | translateSlug: 'GENRES'}}</b>
+                <p>{{ movie.salesAgentDeal.salesAgent.displayName}}</p>
```
Comment (phpgeekfr) [inline apps/catalog/marketplace/marketplace/src/app/movie/search/search.component.html]: conditional test if salesAgent or salesAgent.displayName is undefined ?

File: libs/movie/src/lib/movie/components/display-list/display-list.component.html [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
+    <td mat-cell *matCellDef="let element">{{ element.main.title.original }}</td>
   </ng-container>
 
   <!-- Director column -->
   <ng-container matColumnDef="director">
     <th mat-header-cell *matHeaderCellDef>Director</th>
     <!-- We want only to show one director because of the design right now -->
-    <td mat-cell *matCellDef="let element"> {{element.main.directors[0].firstName}}
-      {{element.main.directors[0].lastName}} </td>
+    <td mat-cell *matCellDef="let element">
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/components/display-list/display-list.component.html]: test directors.length ?

File: libs/ui/src/lib/assets/icons/adjustableWrench.svg [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
     <!-- Generator: Sketch 55.2 (78181) - https://sketchapp.com -->
```
Comment (phpgeekfr) [inline libs/ui/src/lib/assets/icons/adjustableWrench.svg]: I thinks it is `78182` here, not `78181`..

File: libs/ui/src/lib/movie-card/movie-card.component.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
     <!-- Movie Status Chip -->
     <mat-chip *ngIf="movie.main.status as status" selected disableRipple class="movie-chip">
-      <span class="table-size-text">{{ status | translateSlug: 'MOVIE_STATUS'}}</span>
+      <span class="table-size-text">{{ status.toUpperCase() }}</span>
```
Comment (phpgeekfr) [inline libs/ui/src/lib/movie-card/movie-card.component.html]: no more translate slug ?

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html (hunk 1 above)
Comment (ioaNikas) [reply apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: translateSlug pipe is used to get label, here I need to capitalize every letters of the movie status.

File: libs/ui/src/lib/movie-card/movie-card.component.html (hunk 5 above)
Comment (ioaNikas) [reply libs/ui/src/lib/movie-card/movie-card.component.html]: Same as above, movie status is always displayed in uppercase.

**Pull request merged**